app/controllers/activitypub/collections_controller.rb

```diff
@@ -4,12 +4,13 @@ class ActivityPub::CollectionsController < Api::BaseController
   include SignatureVerification
   include AccountOwnedConcern
 
+  before_action :require_signature!, if: :authorized_fetch_mode?
   before_action :set_size
   before_action :set_statuses
   before_action :set_cache_headers
 
   def show
-    expires_in 3.minutes, public: true
+    expires_in 3.minutes, public: !authorized_fetch_mode?
```

Comment:
might be nice to have a readable helper method for `!authorized_fetch_mode?`

Reply:
I'm not sure I agree a simple negation warrants that. `not_authorized_fetch_mode?` would just be longer.